<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiClass</class>
 <widget class="QMainWindow" name="guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>60</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>搜一搜！</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="ResultList">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>441</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="promoteList">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>256</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>推荐结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchNum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchNext">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchPrev">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QLabel" name="promoteNum">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ResultList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>guiClass</receiver>
   <slot>searchResut(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
  <slot>searchResut(QListWidgetItem*)</slot>
 </slots>
</ui>
